src/js/events.js

```diff
@@ -108,6 +108,7 @@ define(function (require, exports, module) {
         tool: {
             SELECT_TOOL_START: "selectToolStart",
             SELECT_TOOL_END: "selectToolEnd",
+            SELECT_TOOL_DRAG: "selectToolDrag",
```

Comment:
This is kind of a confusing name given its proximity to the two aforementioned events `SELECT_TOOL_START` and `SELECT_TOOL_END`, which have a completely unrelated meaning to this new `SELECT_TOOL_DRAG` event...